portality/models/article.py

```diff
@@ -947,6 +951,7 @@ class ArticleIssueQuery(object):
             "bibjson.journal.number",
             "bibjson.title",
             "bibjson.author.name",
+            "bibjson.author.orcid_id",
```

Comment:
It looks to me like this whole class is dead, and should probably be removed along with the function that calls it.  Unless something led you specifically to add this field here?

Reply:
It is used in `get_by_volume_issue` that's why I added that but I haven't checked that this method is not used anywhere else indeed. I'll remove it.